sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -209,6 +208,8 @@ class EthService(val scNodeViewHolderRef: ActorRef, val nvtimeout: FiniteDuratio
       val lowBound = GasUtil.TxGas.subtract(BigInteger.ONE)
       // Determine the highest gas limit can be used during the estimation.
       var highBound = params.gas
+      // we need to set nonce to one, because we substract by one in the evm call
+      params.nonce = BigInteger.ONE
```

Comment:
This will not fix our issue, we need to discuss this issue further.

Reply:
As discussed this might be an issue related to the docker setup and not related to the PR here, so I would suggest closing this PR without any changes regarding the ticket 319.